docs/quick-start.md

```diff
@@ -1,125 +1,11 @@
-Develop your first automations: a bot command and an event handler.
+Develop your first Software Delivery Machine
 
-## Set up
+The easiest way to get started with Atomist is to start on your laptop, working with local commits. Then register your SDM with our service, integrating with Slack, your source control management, and your existing continuous integration tools.
 
-First, be sure someone in your Slack workspace has completed
-the [setup][setup].  Then, install [Node.js][node]
+The latest getting-started tutorials are over at [developer-tutorials](https://github.com/atomist/developer-tutorials/blob/master/README.md) for now. Issues and contributions welcome.
```

Comment:
This README has nothing in it. Better to have the instructions here I would think.

Better to use a named link with a title.

Reply:
will discuss with Lieven tomorrow. Agree that it belongs here. Better to link to it than nothing, right now.
I have a PR in on that repo to put something in the README.